crates/rollup-boost/src/flashblocks/service.rs

```diff
@@ -228,6 +228,14 @@ impl FlashblocksService {
         // consume the best payload and reset the builder
         let payload = {
             let mut builder = self.best_payload.write().await;
+            let flashblocks_number = builder.flashblocks.len();
```

Comment:
I think you can record all the metrics as part of the span and record them there?

Reply:
Good idea, will do it